--- gnome-panel-3.6.2/configure.ac.wiget	2013-03-27 15:54:23.417647031 +0100
+++ gnome-panel-3.6.2/configure.ac	2013-03-27 15:54:37.587048448 +0100
@@ -1,6 +1,6 @@
 AC_INIT([gnome-panel], [3.6.2],
         [http://bugzilla.gnome.org/enter_bug.cgi?product=gnome-panel])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADER(config.h)
 AM_INIT_AUTOMAKE([1.11 no-dist-gzip dist-xz tar-ustar -Wno-portability])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
