diff -aurN gnome-panel-2.11.4.orig/gnome-panel/main.c gnome-panel-2.11.4/gnome-panel/main.c
--- gnome-panel-2.11.4.orig/gnome-panel/main.c	2005-07-02 12:00:31.000000000 +0200
+++ gnome-panel-2.11.4/gnome-panel/main.c	2005-07-05 15:48:13.021834736 +0200
@@ -71,8 +71,6 @@
 	panel_multiscreen_init ();
 	panel_init_stock_icons_and_items ();
 
-	panel_session_init ();
-
 	gconf_client_add_dir (panel_gconf_get_client (),
 			      "/desktop/gnome/interface",
 			      GCONF_CLIENT_PRELOAD_NONE,
@@ -82,6 +80,8 @@
 	panel_lockdown_init ();
 	panel_profile_load ();
 
+	panel_session_init ();
+
 	/*add forbidden lists to ALL panels*/
 	g_slist_foreach (panels,
 			 (GFunc)panel_widget_add_forbidden,
diff -aurN gnome-panel-2.11.4.orig/gnome-panel/panel-session.c gnome-panel-2.11.4/gnome-panel/panel-session.c
--- gnome-panel-2.11.4.orig/gnome-panel/panel-session.c	2005-07-02 12:00:31.000000000 +0200
+++ gnome-panel-2.11.4/gnome-panel/panel-session.c	2005-07-05 15:48:26.584772856 +0200
@@ -74,11 +74,6 @@
 {
 	GnomeClient *client;
 
-	/* We don't want the WM to try and save/restore our
-	 * window position
-	 */
-	gdk_set_sm_client_id (NULL);
-
 	client = gnome_master_client ();
 
         if (!getenv ("GNOME_PANEL_DEBUG"))
