--- gnome-panel-2.91.6/applets/notification_area/Makefile.am.wiget	2011-02-01 00:52:27.000000000 +0100
+++ gnome-panel-2.91.6/applets/notification_area/Makefile.am	2011-02-03 21:57:01.197176223 +0100
@@ -30,16 +30,16 @@ NOTIFICATION_AREA_SOURCES = main.c
 
 NOTIFICATION_AREA_LDADD =				\
 	../../libpanel-applet/libpanel-applet-4.la	\
+	libtray.la					\
 	$(X_LIBS)					\
 	$(NOTIFICATION_AREA_LIBS)			\
-	$(LIBPANEL_APPLET_LIBS)				\
-	libtray.la
+	$(LIBPANEL_APPLET_LIBS)
 
 testtray_SOURCES = testtray.c
 testtray_LDADD =			\
+	libtray.la			\
 	$(X_LIBS)			\
-	$(NOTIFICATION_AREA_LIBS)	\
-	libtray.la
+	$(NOTIFICATION_AREA_LIBS)
 
 if NOTIFICATION_AREA_INPROCESS
 APPLET_IN_PROCESS = true
