diff -Nuard gnome-panel-2.4.1.orig/gnome-panel/panel-stock-icons.c gnome-panel-2.4.1/gnome-panel/panel-stock-icons.c
--- gnome-panel-2.4.1.orig/gnome-panel/panel-stock-icons.c	2003-10-14 12:46:34.000000000 +0200
+++ gnome-panel-2.4.1/gnome-panel/panel-stock-icons.c	2003-11-23 12:57:21.679342744 +0100
@@ -83,7 +83,7 @@
 	{ PANEL_STOCK_SLIDING_PANEL,       "gnome-panel-type-sliding" },
 	{ PANEL_STOCK_MENU_PANEL,          "gnome-panel-type-menu" },
 	{ PANEL_STOCK_LAUNCHER,            "launcher-program" },
-	{ PANEL_STOCK_ACTION,		   NULL },
+	{ PANEL_STOCK_ACTION,		   "gnome-util" },
 	{ PANEL_STOCK_DRAWER,              "panel-drawer" },
 	{ PANEL_STOCK_APPLETS,             "gnome-applets" },
 	{ PANEL_STOCK_DESKTOP,             "gnome-ccdesktop" },
