diff -urN gnome-panel-2.2.0.1.orig/applets/notification_area/eggtraymanager.c gnome-panel-2.2.0.1/applets/notification_area/eggtraymanager.c
--- gnome-panel-2.2.0.1.orig/applets/notification_area/eggtraymanager.c	Tue Feb 25 23:39:22 2003
+++ gnome-panel-2.2.0.1/applets/notification_area/eggtraymanager.c	Tue Feb 25 23:41:34 2003
@@ -399,7 +399,7 @@
 
   invisible = manager->invisible;
   
-  display = GDK_WINDOW_XDISPLAY (manager->invisible);
+  display = GDK_WINDOW_XDISPLAY (manager->invisible->window);
   
   if (XGetSelectionOwner (display, manager->selection_atom) ==
       GDK_WINDOW_XWINDOW (manager->invisible->window))
