--- gnome-panel-2.3.1/configure.in.orig	Sun May  4 12:33:03 2003
+++ gnome-panel-2.3.1/configure.in	Thu May  8 13:45:41 2003
@@ -144,7 +144,7 @@
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE")
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="am ar az be bg ca cs da de el en_GB eo es et eu fa fi fr ga gl he hu it ja ko lt lv mk ml mn ms nl nn no pl pt pt_BR ro ru sk sl sp sq sr sv ta tr uk wa vi yi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg ca cs da de el en_GB eo es et eu fa fi fr ga gl he hu it ja ko lt lv mk ml mn ms nl nn no pl pt pt_BR ro ru sk sl sq sr sr@cyrillic sv ta tr uk wa vi yi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 
