diff -urN gnome-panel-2.10.1.orig/gnome-panel/panel-default-setup.entries.in gnome-panel-2.10.1/gnome-panel/panel-default-setup.entries.in
--- gnome-panel-2.10.1.orig/gnome-panel/panel-default-setup.entries.in	2005-04-05 22:39:34.000000000 +0200
+++ gnome-panel-2.10.1/gnome-panel/panel-default-setup.entries.in	2005-04-05 23:49:24.000000000 +0200
@@ -50,9 +50,6 @@
             <string>window_menu</string>
           </value>
           <value>
-            <string>mixer</string>
-          </value>
-          <value>
             <string>clock</string>
           </value>
           <value>
@@ -370,60 +367,6 @@
     <entry><key>applets/window_menu/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
     <entry><key>applets/window_menu/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
 
-  <!-- Mixer Applet -->
-
-    <entry>
-      <key>applets/mixer/object_type</key>
-      <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
-      <value>
-        <string>bonobo-applet</string>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/mixer/toplevel_id</key>
-      <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
-      <value>
-        <string>top_panel</string>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/mixer/position</key>
-      <schema_key>/schemas/apps/panel/objects/position</schema_key>
-      <value>
-        <int>1</int>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/mixer/panel_right_stick</key>
-      <schema_key>/schemas/apps/panel/objects/panel_right_stick</schema_key>
-      <value>
-        <bool>true</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/mixer/locked</key>
-      <schema_key>/schemas/apps/panel/objects/locked</schema_key>
-      <value>
-        <bool>true</bool>
-      </value>
-    </entry>
-    <entry>
-      <key>applets/mixer/bonobo_iid</key>
-      <schema_key>/schemas/apps/panel/objects/bonobo_iid</schema_key>
-      <value>
-        <string>OAFIID:GNOME_MixerApplet</string>
-      </value>
-    </entry>
-
-    <entry><key>applets/mixer/attached_toplevel_id</key><schema_key>/schemas/apps/panel/objects/attached_toplevel_id</schema_key> </entry>
-    <entry><key>applets/mixer/tooltip</key><schema_key>/schemas/apps/panel/objects/tooltip</schema_key></entry>
-    <entry><key>applets/mixer/use_custom_icon</key><schema_key>/schemas/apps/panel/objects/use_custom_icon</schema_key></entry>
-    <entry><key>applets/mixer/custom_icon</key><schema_key>/schemas/apps/panel/objects/custom_icon</schema_key></entry>
-    <entry><key>applets/mixer/use_menu_path</key><schema_key>/schemas/apps/panel/objects/use_menu_path</schema_key></entry>
-    <entry><key>applets/mixer/menu_path</key><schema_key>/schemas/apps/panel/objects/menu_path</schema_key></entry>
-    <entry><key>applets/mixer/launcher_location</key><schema_key>/schemas/apps/panel/objects/launcher_location</schema_key></entry>
-    <entry><key>applets/mixer/action_type</key><schema_key>/schemas/apps/panel/objects/action_type</schema_key></entry>
-
   <!-- Clock Applet -->
 
     <entry>
