diff -urN gnome-panel-2.9.91.orig/applets/wncklet/workspace-switcher.c gnome-panel-2.9.91/applets/wncklet/workspace-switcher.c
--- gnome-panel-2.9.91.orig/applets/wncklet/workspace-switcher.c	2005-02-28 22:34:58.000000000 +0100
+++ gnome-panel-2.9.91/applets/wncklet/workspace-switcher.c	2005-02-28 22:42:27.750013336 +0100
@@ -212,7 +212,8 @@
 		break;
 	}
 
-	wnck_workspace_activate (wnck_screen_get_workspace (screen, index));
+	wnck_workspace_activate (wnck_screen_get_workspace (screen, index),
+				 event->time);
 	
 	return TRUE;
 }
