--- gnome-panel-2.6.2/gnome-panel/menu.c.wiget	2004-07-21 15:44:04.347155922 +0200
+++ gnome-panel-2.6.2/gnome-panel/menu.c	2004-07-21 15:44:45.256540129 +0200
@@ -221,7 +221,7 @@
 	/*just load the menus from disk, don't make the widgets
 	  this just reads the .desktops of the top most directory
 	  and a level down*/
-	fr_read_dir (NULL, "applications:/", 0, 2);
+	fr_read_dir (NULL, "menu:/", 0, 2);
 
 	if (distribution_info != NULL &&
 	    distribution_info->menu_init_func != NULL)
@@ -2850,12 +2850,12 @@
 {
 	if ( ! fake ||
 	     menu != NULL) {
-		menu = create_menu_at (menu, "applications:/",
+		menu = create_menu_at (menu, "menu:/",
 				       launcher_add,
 				       _("Applications"),
 				       fake_submenus, FALSE);
 	} else {
-		menu = create_fake_menu_at ("applications:/",
+		menu = create_fake_menu_at ("menu:/",
 					    launcher_add,
 					    _("Applications"));
 	}
