--- gnome-panel-2.21.5/applets/clock/Makefile.am	2008-01-15 00:05:22.000000000 +0100
+++ /home/users/patrys/Makefile.am	2008-01-15 13:54:57.436448502 +0100
@@ -9,6 +9,7 @@
 	$(CLOCK_CFLAGS)						\
 	$(POLKIT_GNOME_CFLAGS)					\
 	$(LIBPANEL_APPLET_CFLAGS)				\
+	-DGWEATHER_I_KNOW_THIS_IS_UNSTABLE			\
 	-DDATADIR=\""$(datadir)"\"				\
 	-DGLADEDIR=\""$(datadir)/gnome-panel/glade"\"		\
 	-DICONDIR=\""$(datadir)/gnome-panel/pixmaps"\"		\
@@ -139,6 +140,7 @@
 
 if HAVE_POLKIT
 BUILT_SOURCES += gnome-clock-applet-mechanism.h
+BUILT_SOURCES += gnome-clock-applet-mechanism-glue.h
 endif
 
 gnome_clock_applet_mechanism_INCLUDES =	\
