diff -aurN gnome-panel-2.4.2.orig/gnome-panel/ChangeLog gnome-panel-2.4.2/gnome-panel/ChangeLog
--- gnome-panel-2.4.2.orig/gnome-panel/ChangeLog	2004-01-15 13:48:07.000000000 +0100
+++ gnome-panel-2.4.2/gnome-panel/ChangeLog	2004-01-15 18:40:24.351547832 +0100
@@ -131,6 +131,13 @@
 
 ==================== 2.4.1 ====================
 
+2003-11-28  Marcin Krzyzanowski  <krzak@linux.net.pl>
+
+       * menu.c: (create_applets_menu):
+       Guess untranslated categories based on translated ones, what give
+       us better menu order with translated and untranslated (the same)
+       categories. Fix bug #127732
+
 2003-10-13 Owen Taylor <otaylor@redhat.com>
 
 	* gnome-panel/panel-stock-icons.c (panel_init_stock_icons):
diff -aurN gnome-panel-2.4.2.orig/gnome-panel/menu.c gnome-panel-2.4.2/gnome-panel/menu.c
--- gnome-panel-2.4.2.orig/gnome-panel/menu.c	2003-10-14 12:46:30.000000000 +0200
+++ gnome-panel-2.4.2/gnome-panel/menu.c	2004-01-15 18:41:41.280852800 +0100
@@ -2711,7 +2711,7 @@
 {
 	GtkWidget         *prev_menu = NULL;
 	const char        *prev_category = NULL;
-	int                i;
+	int                i,j,k;
 	const GList       *langs_glist;
 	GSList            *langs_gslist;
 	GSList            *applets = NULL;
@@ -2771,6 +2771,20 @@
 
 		applets = g_slist_append (applets, applet);
 	}
+
+	/* try to guess translations looking for already translated categories */
+	for (j = 0; j < g_slist_length(applets); j++) {
+		AppletMenuInfo	*appletA = (AppletMenuInfo *) g_slist_nth_data (applets,j);
+		
+		if (!g_utf8_collate (appletA->category,appletA->untranslated_category)) {
+			for (k = 0; k < g_slist_length(applets); k++) {
+				AppletMenuInfo	*appletB = (AppletMenuInfo *) g_slist_nth_data (applets,k);
+				
+				if (!g_utf8_collate (appletA->untranslated_category,appletB->untranslated_category))
+					appletA->category = appletB->category;
+			}
+		}
+	}
 	
 	/* add internal action buttons */
 	applets = append_internal_applets (applets);	
