diff -urN gnome-panel-2.32.0.2/applets/notification_area/Makefile.am gnome-panel-2.32.0.2.new//applets/notification_area/Makefile.am
--- gnome-panel-2.32.0.2/applets/notification_area/Makefile.am	2010-09-20 17:49:51.000000000 +0200
+++ gnome-panel-2.32.0.2.new//applets/notification_area/Makefile.am	2010-09-29 22:24:05.220421016 +0200
@@ -30,16 +30,16 @@
 
 NOTIFICATION_AREA_LDADD =				\
 	../../libpanel-applet/libpanel-applet-3.la	\
+	libtray.la					\
 	$(X_LIBS)					\
 	$(NOTIFICATION_AREA_LIBS)			\
-	$(LIBPANEL_APPLET_LIBS)				\
-	libtray.la
+	$(LIBPANEL_APPLET_LIBS)
 
 testtray_SOURCES = testtray.c
 testtray_LDADD =			\
+	libtray.la			\
 	$(X_LIBS)			\
-	$(NOTIFICATION_AREA_LIBS)	\
-	libtray.la
+	$(NOTIFICATION_AREA_LIBS)
 
 if NOTIFICATION_AREA_INPROCESS
 APPLET_IN_PROCESS = true
