diff -urN aa/gnome-panel-2.5.5/gnome-panel/panel-default-setup.entries gnome-panel-2.5.5/gnome-panel/panel-default-setup.entries
--- aa/gnome-panel-2.5.5/gnome-panel/panel-default-setup.entries	2004-01-09 13:56:30.000000000 +0100
+++ gnome-panel-2.5.5/gnome-panel/panel-default-setup.entries	2004-02-17 23:03:27.187869200 +0100
@@ -56,6 +56,9 @@
             <string>clock</string>
           </value>
           <value>
+            <string>notification_area</string>
+          </value>
+          <value>
             <string>show_desktop_button</string>
           </value>
           <value>
@@ -410,6 +413,53 @@
       </value>
     </entry>
   </entrylist>
+	
+  <!-- Notification Area Applet -->
+
+  <entrylist base="/apps/panel/default_setup/applets/notification_area">
+    <entry>
+      <key>object_type</key>
+      <schema_key>/schemas/apps/panel/objects/object_type</schema_key>
+      <value>
+        <string>bonobo-applet</string>
+      </value>
+    </entry>
+    <entry>
+      <key>toplevel_id</key>
+      <schema_key>/schemas/apps/panel/objects/toplevel_id</schema_key>
+      <value>
+        <string>top_panel</string>
+      </value>
+    </entry>
+    <entry>
+      <key>position</key>
+      <schema_key>/schemas/apps/panel/objects/position</schema_key>
+      <value>
+        <int>3</int>
+      </value>
+    </entry>
+    <entry>
+      <key>panel_right_stick</key>
+      <schema_key>/schemas/apps/panel/objects/panel_right_stick_type</schema_key>
+      <value>
+        <bool>true</bool>
+      </value>
+    </entry>
+    <entry>
+      <key>locked</key>
+      <schema_key>/schemas/apps/panel/objects/locked</schema_key>
+      <value>
+        <bool>true</bool>
+      </value>
+    </entry>
+    <entry>
+      <key>bonobo_iid</key>
+      <schema_key>/schemas/apps/panel/objects/bonobo_iid_type</schema_key>
+      <value>
+        <string>OAFIID:GNOME_NotificationAreaApplet</string>
+      </value>
+    </entry>
+  </entrylist>
 
   <!-- Show Desktop Applet -->
 
