--- gnome-panel-2.5.1/applets/clock/clock.c.wiget	2003-11-06 13:50:06.000000000 +0100
+++ gnome-panel-2.5.1/applets/clock/clock.c	2003-11-14 10:31:16.115504722 +0100
@@ -37,6 +37,7 @@
 #include <dirent.h>
 #include <string.h>
 #include <time.h>
+#include <locale.h>
 #include <langinfo.h>
 
 #include <panel-applet.h>
@@ -57,6 +58,8 @@
 
 #define NEVER_SENSITIVE "never_sensitive"
 
+#define FIRST_WEEKDAY_IS_MONDAY 1
+
 static const char* KEY_HOUR_FORMAT   = "hour_format";
 static const char* KEY_SHOW_SECONDS  = "show_seconds";
 static const char* KEY_SHOW_DATE     = "show_date";
@@ -463,6 +466,10 @@
 	GtkWindow *window;
 	GtkWidget *calendar;
 	GtkCalendarDisplayOptions options;
+    gint first_day;
+
+    setlocale (LC_ALL, "");
+    first_day = (int) *(nl_langinfo (_NL_TIME_FIRST_WEEKDAY));
 
 	window = GTK_WINDOW (gtk_window_new (GTK_WINDOW_TOPLEVEL));
 
@@ -482,6 +489,9 @@
 
 	options = gtk_calendar_get_display_options (GTK_CALENDAR (calendar));
 	options |= GTK_CALENDAR_SHOW_WEEK_NUMBERS;
+    if (first_day == FIRST_WEEKDAY_IS_MONDAY) {
+		options |= GTK_CALENDAR_WEEK_START_MONDAY;
+	}
 	gtk_calendar_set_display_options (GTK_CALENDAR (calendar), options);
 
 	gtk_container_add (GTK_CONTAINER (window), calendar);
